pgxn/neon/extension_server.c

```diff
@@ -53,14 +53,13 @@ neon_download_extension_file_http(const char *filename, bool is_library)
     }
 
     compute_ctl_url = psprintf("http://localhost:%d/extension_server/%s%s",
-                      extension_server_port, filename, is_library?"?is_library=true":"");
+                               extension_server_port, filename, is_library ? "?is_library=true" : "");
 
     elog(LOG, "Sending request to compute_ctl: %s", compute_ctl_url);
 
     curl_easy_setopt(curl, CURLOPT_CUSTOMREQUEST, "POST");
     curl_easy_setopt(curl, CURLOPT_URL, compute_ctl_url);
-    curl_easy_setopt(curl, CURLOPT_TIMEOUT, 3L /* seconds */);
-
+    curl_easy_setopt(curl, CURLOPT_TIMEOUT, 100L /* seconds */);
```

Comment:
100 seconds timeout is too large. 
Is it still relevant?

Reply:
I bumped it down to 15seconds. I think for large extensions like postgis this might still be necessary, but we can figure that out later I guess